crates/sui-adapter-transactional-tests/tests/programmable_transaction_examples/upgrade.move

```diff
@@ -0,0 +1,61 @@
+// Copyright (c) Mysten Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+// simple test of upgrade
+
+//# init --addresses p=0x0 q=0x0 q_2=0x0 r=0x0 s=0x0 --accounts A
+
+//# publish
+module p::m {
+    public fun foo(x: u64) {
+        p::n::bar(x)
+    }
+}
+module p::n {
+    public fun bar(x: u64) {
+        assert!(x == 1, 0);
+    }
+}
+
+//# publish --upgradeable --sender A
+module q::m {
+    public fun x(): u64 { 0 }
+}
+
+//# publish
+module r::m {
+    public fun y(): u64 { 0 }
+}
+
+//# package
```

Comment:
Thoughts on naming this `stage-package` to match the command type name? Or `declare-package` or some such? Just a minor sense that `package` makes it a little less obvious what this does or corresponds to in code

Reply:
I like `stage-package`, will change 👍